### ISSUE TITLE
[Android][WebView] WebViewConfig is useless

### ISSUE THREAD

**Issue by maarekj** [facebook/react-native#9046]
Hi,
The ReactWebViewManager, in Android, use a WebViewConfig to configure the webview when it is created. *(As it says in the [following file](https://github.com/facebook/react-native/blob/a2fb703bbb988038323c55b29b40e8f5ff52966d/Libraries/Components/WebView/WebView.android.js#L141))*

On the other hand, it is not possible of to change the WebViewConfig, because the ReactWebViewManager has not setter for this property. And the property mWebViewConfig is private therefore impossible to change it.

**TL;DR**
Is it possible to add a setter method for change WebViewConfig in ReactWebViewManager ?

Thanks.

**Comment by maarekj** [comment]
Sorry I didn't see the constructor with param WebViewConfig.

**Comment by maarekj** [comment]
Even if the constructor allow to pass a WebViewConfig, it would be useful to change it the a setter method. No ?

**Comment by lacker** [comment]
This looks stale, and also we are encouraging issues for bugs and this sort of general how-to question may be better for Stack Overflow.

**Comment by SoundBlaster** [comment]
@laker This is not a stale, this is a very actual issue.
`ReactWebView` contains method `createViewInstance` that uses instance of `WebViewConfig` in `mWebViewConfig` variable to prepare created instance of `ReactWebView` by custom logic of configurator if it was previously passed to constructor of 'ReactWebViewManager'. You can find all code [here](https://github.com/facebook/react-native/blob/d2f0abdf4ea94fbb3e2a5c7fb53ff5d1cf6abede/ReactAndroid/src/main/java/com/facebook/react/views/webview/ReactWebViewManager.java).

As an example of potential usage you can see [this commit](https://github.com/SoundBlaster/react-native/commit/613e732be0a6b1001d863630637b3f3d932dec94) in my old fork of RN. I need to clean cache of WebView on Android and if `WebView` had a public API to customise a `WebViewConfig` it was great. But it wasn't.

**Comment by SoundBlaster** [comment]
@maarekj can you support my opinion?
